charts/minecraft/templates/rcon-svc.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   name: "{{ template "minecraft.fullname" . }}-rcon"
   annotations:
     {{ toYaml .Values.minecraftServer.serviceAnnotations | indent 4 }}
+    {{ toYaml .Values.serviceAnnotations | indent 4 }}
```

Comment:
Do you think we will need a unique `serviceAnnotation` for both the regular service as well as the rcon service, or is combining them ok?

Reply:
Yeah, I'm not sure either way.

Reply:
FWIW, I use this to add a metallb annotation to allow those multiple services to share the same LoadBalancer application. I can't give any insight on possible use case to set this separately, but I'd assume there are, and allowing a separate config might be the the most future proof option?